aws_lambda_builders/workflows/nodejs_npm/actions.py

```diff
@@ -0,0 +1,110 @@
+"""
+Action to resolve NodeJS dependencies using NPM
+"""
+
+import logging
+from aws_lambda_builders.actions import BaseAction, Purpose, ActionFailedError
+from .npm import NpmExecutionError
+
+LOG = logging.getLogger(__name__)
+
+
+class NodejsNpmPackAction(BaseAction):
+
+    """
+    A Lambda Builder Action that packages a Node.js package using NPM to extract the source and remove test resources
+    """
+
+    NAME = 'NpmPack'
+    DESCRIPTION = "Packaging source using NPM"
+    PURPOSE = Purpose.COPY_SOURCE
+
+    def __init__(self, artifacts_dir, scratch_dir, manifest_path, osutils, subprocess_npm):
```

Comment:
We are inheriting but not calling super. Would be a good practice to start especially with this being a library.

Reply:
ok, resolved with https://github.com/gojko/aws-lambda-builders/commit/53f9d349da988abf445ce06b4d291240c69bbb7c